doctrine.rst

```diff
@@ -820,6 +820,21 @@ control behavior:
 ``disabled``
     If true, the ``EntityValueResolver`` will not try to replace the argument.
 
+``message``
+    If a ``message`` option is configured, the value of the ``message`` option will be displayed
```

Comment:
The word "displayed" may be confusing here, leading to think the message will be displayed on the error page (it will not)

See https://github.com/symfony/symfony/pull/51562#issuecomment-1705990607

Maybe "used" or "set" would be better ?

Reply:
Yes or "will be displayed in the development environment", what do you think?